boostregex/UTF8DocumentIterator.cxx

```diff
@@ -58,6 +62,27 @@ UTF8DocumentIterator::UTF8DocumentIterator(const UTF8DocumentIterator& copy) :
 		}
 }
 
+UTF8DocumentIterator& UTF8DocumentIterator::operator ++ ()

```

Comment:
But why is it needed?

Reply:
Because I didn't want to add 
#include <string_view>
#include <stdexcept>
for
#include "Platform.h"
in the header file.